environments/dev-env/CONTRIBUTING.md

```diff
@@ -0,0 +1,510 @@
+# KCP-Edge Contributor Development Environment
+
+## Required Packages:
+   - ko: https://ko.build/install/ 
+   - gcc
+   - docker 
+   - jq
+   - make
+   - go (version expected 1.19)
+   - kind
+   - kubectl  
+
+
+## Supported OS Platforms 
+  - Linux
+  - MacOS
+  - Windows WSL/Ubuntu
+
+
+Run the following script to install the required package (Linux or MacOS ):
+
+```bash
+./install_req.sh
+```
+
+Alternatively, run the following command for MacOS platform:
+
+```bash
+brew install ko gcc jq make go@1.19 kind kubectl
+```
+
+For Windows WSL/Ubuntu platform, follow the instructions [here](docs/wls_ubuntu_platform.md)
+
+
+## Quickstart
+
+### 1. Clone the kcp-edge repo:
+
+```shell
+git clone https://github.com/kcp-dev/edge-mc.git  kcp-edge
+```
+
+### 2. Build the kcp-edge binaries:
+
+```shell
+cd kcp-edge
+make build
+```
+
+The kcp-edge binaries is created inside the `/bin` directory. Add this directory to your $PATH:
+
+```shell
+export PATH="$PATH:$(pwd)/bin"
+```
+
+
+### 3. Create your own edge infrastructure (edge clusters) - Bring Your Own Cluster (BYOC)
+
+Create your edge cluster or bring your own k8s edge cluster. In this example, we use [kind](https://kind.sigs.k8s.io/) to create an edge cluster that we name “florin”:
+
+```shell
+kind create cluster --name florin
+```  
+
+### 4. Deploy the kcp-edge platform:
+
+  a) Download kcp binaries for your platform:
+
+  ```shell
+  mkdir kcp
+  cd kcp
+  VERSION=0.11.0 # choose the latest version (without v prefix)
+  OS=darwin   # linux or darwin
+  ARCH=amd64  # arm64 or amd64
+  curl -sSfL "https://github.com/kcp-dev/kcp/releases/download/v${VERSION}/kcp_${VERSION}_${OS}_${ARCH}.tar.gz" > kcp.tar.gz
+  curl -sSfL "https://github.com/kcp-dev/kcp/releases/download/v${VERSION}/kubectl-kcp-plugin_${VERSION}_${OS}_${ARCH}.tar.gz" > kubectl-kcp-plugin.tar.gz
+  ```
+
+  Extract kcp and kubectl-kcp-plugin and place all the files in the bin directories somewhere in your $PATH. For example:
+
+  ```shell
+  tar -xvf kcp.tar.gz
+  tar -xvf kubectl-kcp-plugin.tar.gz
+  export PATH="$PATH:$(pwd)/bin"
+  ```
+
+  b) Start kcp:
+
+  ```shell
+  kcp start >& kcp_log.txt &
+  ```
+  OR
+  
+  If you need to connect to a remote k8s edge cluster, you can bind kcp to the public interface of your virtual machine:
+
+  ```shell
+  kcp start --bind-address <ip-address> >& kcp_log.txt &
+  ```
+  
+  This uses default 6443 port, just bind to a specific address
```

Comment:
I suggest the following revision, for section (b) regarding starting kcp.

b) Start kcp:

```shell
kcp start --bind-address <ip-address> >& kcp_log.txt &
```

In that command, `<ip-address>` is an IP address of the machine running kcp that can be reached from the edge clusters.  This is the address that will appear in the generated kubeconfig file (mentioned next) and the kcp server will listen only at that address.  If all of the host's non-loopback addresses can be reached from the edge clusters then this is a non-issue and the `--bind-address <ip-address>` can be omitted from the command line; in this case the server will listen at all addresses and pick a non-loopback one to put in the generated kubeconfig file.

Reply:
Perhaps `kcp start [--bind-address <ip-address>] >& kcp_log.txt &`

Reply:
That works for me.